core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -824,4 +824,51 @@ class PlaintextConsumerTest extends BaseConsumerTest {
 
     assertThrows(classOf[WakeupException], () => consumer.position(topicPartition, Duration.ofSeconds(100)))
   }
+
+  // NOTE: at present this test fails for the CLASSIC group protocol when run on the Apache CI infrastructure. For
+  //       now this will remain a test that only covers the CONSUMER group protocol.
```

Comment:
Can we enable this test for `ClassicKafkaConsumer` since we use github CI recently and it pass on my local?

Reply:
Good idea. I'll give it a go.